pages/project-registry/roles.md

```diff
@@ -1,8 +1,66 @@
 # Roles
+## Intro
+The registry provides a simple role system that can be used by projects integrating with the registry.
+
+There are two roles available, owner and member.
+## Owner
+### Privileges
+The owner is the most privileged role in a profile. There can only be one owner
+at a time on a profile. 
+Only the owner address has the ability to:
+
+* Update the profile metadata
+```
+updateProfileMetadata(bytes32 _profileId, Metadata memory _metadata)
+```
+* Add or remove profile members: 
+```
+addMembers(bytes32 _profileId, address[] memory _members)
+removeMembers(bytes32 _profileId, address[] memory _members)
+```
+* Change the profile name 
+```
+// Note that this call will also update the profile anchor
+updateProfileName(bytes32 _profileId, string memory _name)
+```
+
+* Transfer ownership to a new address
+```
+updateProfilePendingOwner(bytes32 _profileId, address _pendingOwner)
+```
+### Setting the Owner
+The owner is set to msg.sender when the profile is created.
```

Comment:
[Turns out this is not the case](https://github.com/allo-protocol/allo-v2/blob/main/contracts/core/Registry.sol#L80-L86)

This could end up changing though so that it is the case